### PR TITLE
Deprecate sip21

### PR DESCRIPTION
**Pull request by hav-noms**

### CODE REVIEW

File: contracts/Exchanger.sol [hunk 1]
```diff
@@ -61,19 +61,7 @@ contract Exchanger is MixinResolver {
         // Get the base exchange fee rate
         uint exchangeFeeRate = feePool().exchangeFeeRate();
 
-        uint multiplier = 1;
```
Comment (justinjmoses) [inline contracts/Exchanger.sol]: Qu: do we still need this function at all then? Or should we just always use it from `FeePool`?

File: test/contracts/Exchanger.js [hunk 2]
```diff
@@ -1342,7 +1324,7 @@ contract('Exchanger (via Synthetix)', async accounts => {
 											from: iBTC,
 											to: sAUD,
 											toContract: sAUDContract,
-											exchangeFeeRateMultiplier: 2,
+											exchangeFeeRateMultiplier: 1,
```
Comment (justinjmoses) [inline test/contracts/Exchanger.js]: You can safely remove this prop

File: contracts/Exchanger.sol (hunk 1 above)
Comment (hav-noms) [reply contracts/Exchanger.sol]: We might need this for differential fees

Comment (justinjmoses) [reply contracts/Exchanger.sol]: Gotcha

**Pull request merged**